lib/chore/queues/filesystem/publisher.rb

```diff
@@ -40,7 +33,8 @@ def publish(queue_name,job)
         def filename(queue_name, job_name)
           now = Time.now.strftime "%Y%m%d-%H%M%S-%6N"
           previous_attempts = 0
-          File.join(new_dir(queue_name), "#{queue_name}-#{job_name}-#{now}.#{previous_attempts}.job")
+          pid = Process.pid
+          File.join(new_dir(queue_name), "#{queue_name}-#{job_name}-#{pid}-#{now}.#{previous_attempts}.job")
```

Comment:
Non blocking nit: I'm not currently seeing where we increment `previous_attempts` anywhere, so does it make sense to have it in the filename?

Reply:
It gets incremented by the consumer when there's an error.  See https://github.com/Tapjoy/chore/blob/master/lib/chore/queues/filesystem/consumer.rb#L88

Reply:
👍  carry on.